turbopy/core.py

```diff
@@ -419,6 +433,14 @@ def initialize(self):
         """
         pass
 
+    def __repr__(self):
+        representation = f"{self.__class__.__name__}{self.module_type}, {self.input_data})"
+        return representation
+
+    def __str__(self):
+        base_str = f"A PhysicsModule with name {self.module_type}."
+        return base_str
```

Comment:
There is a parenthesis missing in `__repr__`. Also, `module_type` is not necessarily defined (i.e., could be `None`).
```suggestion
    def __repr__(self):
        return f"{self.__class__.__name__}({self.input_data})"
```

I don't think we need `__str__` right now.